html-formatter/ruby/Makefile

```diff
@@ -0,0 +1,42 @@
+include default.mk
+# https://stackoverflow.com/questions/2483182/recursive-wildcards-in-gnu-make
+rwildcard=$(foreach d,$(wildcard $(1:=/*)),$(call rwildcard,$d,$2) $(filter $(subst *,%,$2),$d))
+
+SOURCE_FEATURES=$(call rwildcard,../javascript/features,*.feature)
+FEATURES=$(patsubst ../javascript/%,%,$(SOURCE_FEATURES))
+CCK_MESSAGES=$(call rwildcard,../../compatibility-kit/ruby,*.ndjson)
+
+.deps: assets/cucumber-react.css assets/cucumber-html.js assets/index.mustache.html
+
+.tested: acceptance/cck.html
+
+acceptance/cck.html: $(CCK_MESSAGES)
+	mkdir -p $(@D)
+	cat $^ | bundle exec bin/cucumber-html-formatter --format ndjson > $@
+
+acceptance/cucumber.html: .deps $(FEATURES)
+	mkdir -p $(@D)
+	../../fake-cucumber/javascript/bin/fake-cucumber \
+	  --format ndjson \
+		features/*.feature | \
+		bundle exec bin/cucumber-html-formatter --format ndjson > \
+		$@
+
+features/%.feature: ../javascript/features/%.feature
+	cp $^ $@
+
+assets/cucumber-react.css:
+	mkdir -p $(@D)
+	cp ../../react/javascript/dist/src/styles/cucumber-react.css $@
+
+assets/cucumber-html.js:
+	mkdir -p $(@D)
+	cp ../javascript/dist/main.js $@
+
+assets/index.mustache.html:
+	mkdir -p $(@D)
+	cp ../javascript/dist/src/index.mustache.html $@
+
+
+clean:
```

Comment:
Does this override the `clean` directive in the included make file or does it extend it?

Reply:
it extends it:

```shell
╭─vincent.pretre@Vincents-MacBook-Pro ~/dev/open-source/cucumber/html-formatter/ruby ‹ruby-2.6.3› ‹html-formatter-for-ruby› 
╰─$ make clean
rm -f .deps .linked .tested* Gemfile.lock *.gem
rm -rf acceptance features/all_green.feature features/statuses.feature assets/*.css assets/*.js assets/*.html
```

First line comes from `default.mk`, second one from `Makefile`

Reply:
Cool. Still a bit hazy on how includes actually work.